pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Managing schemas using Schema Registry
+description: A tutorial teaching how to manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+In this tutorial, you'll learn how to manage schemas with Schema Registry. Managing schemas ensures a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains schemas for data exchanged between various components of a modern distributed system. Validating exchanged data maintains data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the [Apicurio Registry](https://www.apicur.io/registry/) for this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will use Apicurio Registry combined with Avro, a language-neutral data serialization system.
+
+## Prerequisites
+[Install Docker](https://docs.docker.com/engine/install/) from the official website.
+
+
+### Start Apicurio Registry
+Start the Apicurio Registry locally with the following docker command:
+   
+```
+{`docker run --env CORS_ALLOWED_ORIGINS=* -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.8.Final`}
```

Comment:
Can you check it again?

```
➜  ~ docker run --env CORS_ALLOWED_ORIGINS=* -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.8.Final
zsh: no matches found: CORS_ALLOWED_ORIGINS=*
```

Maybe problem only on my side, but looks like value must be escaped with `'`

```
➜  ~ docker run --env CORS_ALLOWED_ORIGINS='*' -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.8.Final
Starting the Java application using /opt/jboss/container/java/run/run-java.sh ...
INFO exec -a "java" java -Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -cp "." -jar /deployments/apicurio-registry-app-2.5.8.Final-runner.jar
INFO running in /deployments
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
```

Reply:
nit: I don't now how command will be rendered on out site, but I'm can't understand why here are `{` with `'` around command

Reply:
@Arya-Gupta feedback to check out ⬆️

Reply:
Lukasz sent me this, I'm not totally sure about which version is correct. I'm going to take help from another contributor and try running both versions of the command.

Reply:
> Lukasz sent me this, I'm not totally sure about which version is correct. I'm going to take help from another contributor and try running both versions of the command.

You can use command, where I wrapped *. Now works correct

Reply:
@Pakisan I've modified it!